derivative.tex

```diff
@@ -1373,1305 +1378,1344 @@
+	\end{change}
+	\smallskip
+	\begin{change}[version = 0.97, date = 2020-02-03, beta = true]
+		\item Fixed the argument specifier of {\small\verb|\__deriv_scale_big:nnnn|} when it was used (it was used with \texttt{:nnnm}).
+	\end{change}
+	\smallskip
+	\begin{change}[version = 0.98, date = 2020-07-20, beta = true]
+		\item Fixed a bug concerning the subscript argument due to recent changes to the \pkg{xparse} package dated 2020-05-14 (the fix works with earlier versions of \pkg{xparse} as well).
+	\end{change}
+	\begin{change}[version = 1.0, date = 2021-05-25, beta = false]
+		\item Added new options for derivatives
+		\item Added new values for existing options
+		\item Added new commands to write differentials
+		\item Changed usage of \key[index=false]{style-inf}
+		\item Changed the order argument to an option argument
+		\item Changed default values for some options
+		\item Changed math space keys to accept more general inputs
+		\item Changed \cs{derivset} to define default options
+		\item Merged the codes for ordinary and partial derivatives
+		\item Removed the mixed order argument
+		\item Removed the options \key[index=false]{misc-add-delims} and \key[index=false]{misc-remove-delims}
+		\item Replaced the commands in section 5 with new ones. Old ones no longer works.
+		\item Code clean-up and optimization
+		\item Fixed code errors
+		\item Fixed documentation errors
+		\item Enhanced support for preventing options effecting nested derivatives/differentials
+	\end{change}
+	\begin{change}[version = 1.01, date = 2021-05-28, beta = false]
+		\item Changed the option \key[index=false]{sep-end} default value because it caused issues
+		\item Fixed code errors due to changes introduced in version 1.0 to the option \key[index=false]{style-inf}
+		\item Fixed documentation errors
+	\end{change}
+	\begin{change}[version = 1.1, date = 2021-06-03, beta = false]
+		\item Added package options to be used in \cs{usepackage}\oarg{options}\narg{derivative}
+		\item No longer secretly depends on \pkg{unicode-math} when using \xetex{} or \luatex
+		\item The package no longer uses \cs{symup} which have been replaced with \cs{mathrm} for simplicity
+		\item If \pkg{mleftright} is loaded then the default value of \key[index=false]{scale-auto} is \val{mleftmright}
+	\end{change}
+	\begin{change}[version = 1.2, date = 2022-07-09, beta = false]
+		\item Code clean-up and optimisation that significantly speeds up usage of the package
+		\item Added new sorting algorithm \keyval[index=false]{sort-method}{lexical} that sorts terms alphabetically
+		\item Added the option \key[index=false]{sort-lexical-reverse} 
+		\item Added the option \key[index=false]{switch-sort} that turns on/off the sorting algorithm of the mixed order
+		\item Fixed bug concerning \keyval[index=false]{scale-\meta{\dots}}{none} when the corresponding delimiter option contained a period.
+		\item Fixed documentation errors
+	\end{change}
+\end{changelog}

```

Comment:
I think we add things about `style-inf-denom` and `switch-compact`.

I can do it if you want.